lib/ui/index.js

```diff
@@ -190,6 +190,7 @@ class UI {
         cp.stderr.on('data', (data) => {
             const lines = data.toString('utf8').split('\n');
 
+            /* istanbul ignore else */
```

Comment:
Not adding a test here because (for now) we don't care about other command output. There's no point in checking if the sudo password _isn't_ asked for, is there?